nomos-services/sdp/src/adapters/mempool/mock.rs

```diff
@@ -0,0 +1,26 @@
+use nomos_core::mantle::{SignedMantleTx, Transaction as _};
+use overwatch::DynError;
+
+use super::SdpMempoolAdapter;
+
+pub struct MockMempoolAdapter;
+
+#[async_trait::async_trait]
+impl SdpMempoolAdapter for MockMempoolAdapter {
+    fn new() -> Self {
+        Self {}
+    }
+
+    async fn post_tx(&self, tx: SignedMantleTx) -> Result<(), DynError> {
+        // TODO: enable when membership service is deleted and we can depend on
```

Comment:
@bacv Does this makes sense? I was getting a circular dependency when trying to implement a real adapter. I think because of tx_service->da-sampling -> da-network -> membership -> sdp, and now we have da-network -> sdp. 

But it looks like this will be ok when membership service is removed, and then we can add the real adapter implementation.